images/crossplane-aws/main.tf

```diff
@@ -47,9 +47,21 @@ module "test-latest" {
   digests = { for k, v in module.latest : k => v.image_ref }
 }
 
-resource "oci_tag" "latest" {
-  for_each   = toset(local.components)
+module "version-tags" {
+  for_each = toset(local.components)
+  source   = "../../tflib/version-tags"
+  package  = "crossplane-provider-aws-${each.key}"
+  config   = module.latest[each.key].config
+}
```

Comment:
I think this should live downstream.  I don't think we should be adding more `version-tags` in `images`

Reply:
I'm fine with that. However a customer is waiting on this and I'd like to ask to clean this up later, with the rest.